docker-for-windows/wsl-tech-preview.md

```diff
@@ -1,55 +1,92 @@
+Ensure you have completed the steps described in the Prerequisites section **before** installing the Docker Desktop Edge 2.2.3.0 release.
 
-# Install
-
-Ensure you have completed the steps described in the Prerequisites section **before** installing the Docker Desktop Edge release.
-
-1. Follow the usual Docker Desktop installation instructions to install Docker Desktop.
+1. Follow the usual installation instructions to install Docker Desktop. If you are running a supported system, Docker Desktop prompts you to enable WSL 2 during installation. Read the information displayed on the screen and enable WSL 2 to continue.
 2. Start Docker Desktop from the Windows Start menu.
 3. From the Docker menu, select **Settings** > **General**.
 
-    ![WSL 2 Tech Preview Desktop UI](images/wsl2-enable.png)
+    ![Enable WSL 2](images/wsl2-enable.png){:width="750px"}
+
+4. Select the **Use WSL 2 based engine** check box.
 
-4. Select the **Enable the experimental WSL 2 based engine** check box.
+    If you have installed Docker Desktop on a system that supports WSL 2, this option will be enabled by default.
 5. Click **Apply & Restart**.
 6. Ensure the distribution runs in WSL 2 mode. WSL can run distributions in both v1 or v2 mode.
 
-    To check the WSL mode, run:
+    To check the WSL mode, run
+
+     `wsl -l -v`
+
+    To upgrade your existing Linux distro to v2, run:
+
+    `wsl --set-version (distro name) 2`
+
+    To set v2 as the default version for future installations, run:
+
+    `wsl --set-default-version 2`
+
+7. When Docker Desktop restarts, go to **Settings** > **Resources** > **WSL Integration**.
+
+    WSL Integration will be enabled on your default WSL distribution. To change your default WSL distro, run `wsl --set-default <distro name>`.
+
+    For example, to set Ubuntu as your default WSL distro, run `wsl --set-default ubuntu`.
+
+    Optionally, select any additional distributions you would like to enable WSL 2 on.
+
+    ![WSL 2 Choose Linux distro](images/wsl2-choose-distro.png)
+
+8. Click **Apply & Restart**.
+
+## Develop with Docker and WSL 2
+
+The following section describes how to start developing your applications using Docker and WSL 2. We recommend that you have your code in your default Linux distribution for the best development experience using Docker and WSL 2. After you have enabled WSL 2 on Docker Desktop, you can start working with your code inside the Linux distro and ideally with your IDE still in Windows. This workflow can be pretty straightforward if you are using [VSCode](https://code.visualstudio.com/download).
+
+1. Open VSCode and install the [Remote - WSL](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) extension. This extension allows you to work with a remote server in the Linux distro and your IDE client still on Windows.
+2. Now, you can start working in VSCode remotely. To do this, open your terminal and type:
+
+    `Wsl`
```

Comment:
I don't have a WSL-capable computer I can test this on, but does this really have an upper case `W`?

Reply:
@simonferquel Could you confirm this? I got this command from the blog - https://www.docker.com/blog/docker-desktop-for-windows-home-is-here/

Reply:
On Windows, executables are case-insensitive. But `wsl.exe`  feels better than `Wsl.exe`.

Reply:
Great, thanks for confirming.